cloudnet-modules/cloudnet-bridge/src/main/java/de/dytanic/cloudnet/ext/bridge/node/listener/NodeCustomChannelMessageListener.java

```diff
@@ -45,22 +46,30 @@ public void handle(ChannelMessageReceiveEvent event)
             {
                 NetworkConnectionInfo networkConnectionInfo = event.getData().get("networkConnectionInfo", NetworkConnectionInfo.TYPE);
                 CloudNetDriver.getInstance().getEventManager().callEvent(new BridgeProxyPlayerLoginRequestEvent(networkConnectionInfo));
-                System.out.println(LanguageManager.getMessage("module-bridge-player-proxy-login-request")
-                    .replace("%uniqueId%", networkConnectionInfo.getUniqueId() + "")
-                    .replace("%name%", networkConnectionInfo.getName() + "")
-                    .replace("%proxy%", networkConnectionInfo.getNetworkService().getServerName() + "")
-                );
+
+                BridgeConfiguration bridgeConfiguration = CloudNetBridgeModule.getInstance().getBridgeConfiguration();
```

Comment:
Please only take the Conifguration once and make it a class variable, in the constructor would be good